<launch>
    <!--<node pkg="rosserial_python" type="serial_node.py" name="serial_node">-->
        <!--<param name="port" value="/dev/ttyUSB0"/>-->
        <!--<param name="baud" value="115200"/>-->
    <!--</node>-->

    <include
        file="$(find aist)/launch/basic.launch"
        clear_params="false"
        pass_all_args="true"/>

    <!--<node pkg="aist" type="movement_node" name="movement_node" output="screen" />-->
    <node pkg="aist" type="movement_node" name="movement_node" />

</launch>